src/zodbupdate/serialize.py

```diff
@@ -298,18 +293,17 @@ def rename(self, input_file):
         return None otherwise.
         """
         self.__changed = False
-        self.__skipped = False
 
         unpickler = self.__unpickler(input_file)
         class_meta = unpickler.load()
         data = unpickler.load()
 
-        class_meta = self.__update_class_meta(class_meta)
-
-        if self.__skipped:
+        if class_meta is Blob:
```

Comment:
Sadly enough, life is not that simple. That was the original implementation proposed here, but ``class_meta`` can be more than just a class itself, it can be a tuple, which itself has different formats. That's why I tried to do something with this ``self.__skipped`` flag, because the rewrite mechanism will out of all possible format always find the symbol that leads to the class.

I do believe that for recent database, that would indeed work for blobs, but I would rather properly check all possible formats.

As well I believe there's value in having this skip more generic. We should maybe take the ignore list as a parameter in the ``__init__``.